### PR TITLE
Bump LTS 1.8 version to 1.8.2 and hardcode builder branch

### PR DESCRIPTION
**Pull request by antoniovs1029**
We'd want binaries produced with the `ci/binaries` label to have 1.8.2 as version. @seemethere @malfet

### CODE REVIEW

**Review by malfet — APPROVED** [review]
LGTM

File: .circleci/scripts/binary_checkout.sh [hunk 1]
```diff
@@ -63,6 +63,8 @@ popd
 # Clone the Builder master repo
 retry git clone -q https://github.com/pytorch/builder.git "$BUILDER_ROOT"
 pushd "$BUILDER_ROOT"
+git fetch
```
Comment (antoniovs1029) [inline .circleci/scripts/binary_checkout.sh:66]: @malfet @seemethere , this is the only place where I need to point to the correct builder/ branch, right?

File: .circleci/scripts/binary_checkout.sh [hunk 2]
```diff
@@ -63,6 +63,8 @@ popd
...
+git checkout lts/release/1.8
```
Comment (malfet) [inline .circleci/scripts/binary_checkout.sh:67]: Nit: one can specify branch to clone from directly as part of `git clone` command
```suggestion
retry git clone -q https://github.com/pytorch/builder.git -b lts/release/1.8 "$BUILDER_ROOT"
pushd "$BUILDER_ROOT"
```

File: .circleci/scripts/binary_checkout.sh (hunk 1 above)
Comment (malfet) [reply .circleci/scripts/binary_checkout.sh]: I suggest we pick the branch explicitly while cloning.

But yes, judging by quick grepping this is the only place

**Comment by harishsk** [comment]
@facebookgithubbot retest me please

**Pull request merged**